provisioning_virtual_machines_and_hosts/_topics/provisioning_requests.md

```diff
@@ -661,7 +661,7 @@ marked with **Use cloud-init**; although, the template from which the virtual ma
 
 **Note:**
 
-For an example of virtual machine provisioning request using cloud-init via REST API, see *Provisioning a Virtual Machine Using Cloud-init* in the [{{ site.data.product.title_short }} REST API Guide](../rest_api/index.html#provisioning-a-virtual-machine-using-cloud-init) guide.
+For an example of virtual machine provisioning request using cloud-init via REST API, see *Provision Requests* in the [{{ site.data.product.title_short }} API Guide](..api/reference/provision_requests.html) guide.
```

Comment:
Should this be `../api` ? (missing slash

Reply:
Yes, missed that /